editor/src/clj/editor/gl/shader.clj

```diff
@@ -544,7 +544,8 @@ of GLSL strings and returns an object that satisfies GlBind and GlEnable."
       (aget out-uniform-count 0))))
 
 (def ^:private uniform-info
-  (let [name-buffer-size 128
+  (let [name-array-suffix-pattern #"\[\d+\]$"
+        name-buffer-size 128
```

Comment:
OpenGL was reporting array uniforms including a "[0]" suffix to the name. When we apply uniforms from materials, we want to address the uniform by name without this "[0]" suffix.

Here, we strip out the array suffix from the name we associate with the uniform in the editor. This behavior mirrors what we do in the runtime, since users can set uniforms from scripts as well.